Mathlib/Topology/LocallyConstant/Basic.lean

```diff
@@ -592,15 +592,15 @@ section Equiv
     homeomorphism `X ≃ₜ Y` -/
 noncomputable
```

Comment:
```suggestion
@[simps]
noncomputable
```

You can check the lemmas generated by `@[simps]` on `congrLeftₗ` before and after this change to see the difference!